nats-subscriber/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
```

Comment:
We may need to specify the files you're copying in or use a .dockerignore in this directory to avoid copying in the README in this directory. While that's not a show stopper for this image, it's something to keep in mind for future images, and honestly I would need to go back to see if I used a .dockerignore for the existing images as well.

Reply:
good idea.  easy enough to test.

Reply:
Actually, a .dockerignore for the README will make it difficult to update the README later.

Reply:
Handled by the new dockerfile - only the .js and package.json are copied to the image